docs/edge.rst

```diff
@@ -48,29 +48,32 @@ Kubernetes cluster, one that meets the following requirements:
 
 .. rst-class:: open
 
-* It must contain at least three nodes (for high availability);
+* It must contain at least three nodes (for high availability).
+  You can also run development workloads on a single-node cluster, however please note
+  that you will only be able to provision single-node CrateDB "clusters";
```

Comment:
I'm not sure I understand this - what is the 'however' here?

Reply:
I think you changed that in one of the suggestions and I applied it.